Docs/Configuration.tex

```diff
@@ -2766,126 +2775,135 @@ \section{Misc}\label{misc}
-  \item If partition handle is marked as \textit{unused} execute ``bless'' primary option list retrieval. \\
-    In case \texttt{BlessOverride} list is set, not only standard ``bless'' paths will be found but
-    also custom ones.
-  \item On OpenCore boot partition exclude all OpenCore bootstrap files by header checks.
+  \item If the partition handle is marked as \textit{unused}, execute ``bless'' primary option list retrieval. \\
+    In case \texttt{BlessOverride} list is set, both standard and custom ``bless'' paths will be found.
+  \item On the OpenCore boot partition, exclude OpenCore bootstrap files using header checks.
   \item Register the resulting entries as primary options and determine their types if found. \\
   The option will become auxiliary for some types (e.g. Apple HFS recovery).
   % Looking up primary and alternate handles could be done per handle to make sure the list is ordered.
-  \item If partition already has primary options of ``Apple Recovery'' type proceed to next handle.
+  \item If a partition already has any primary options of the ``Apple Recovery'' type, proceed to the next handle.
   \item Lookup alternate entries by ``bless'' recovery option list retrieval and predefined paths.
   \item Register the resulting entries as alternate auxiliary options and determine their types if found.
   \end{itemize}
 \item Custom entries and tools are added as primary options without any checks with respect to \texttt{Auxiliary}.
-\item System entries (e.g. \texttt{Reset NVRAM}) are added as primary auxiliary options.
+\item System entries, such as \texttt{Reset NVRAM}, are added as primary auxiliary options.
 \end{enumerate}
 
-The display order of the boot options in the picker and the boot process are determined separately from the scanning
-algorithm. The display order as follows:
+The display order of the boot options in the boot picker and the boot process are determined separately from the scanning
+algorithm.
+
+The display order as follows:
 
 \begin{itemize}
 \tightlist
-\item Alternate options follow corresponding primary options, i.e. Apple recovery will be following the
+\item Alternate options follow corresponding primary options. That is, Apple recovery options will follow the
   relevant macOS option whenever possible.
-\item Options will be listed in file system handle firmware order to maintain an established order across
-  the reboots regardless of the chosen operating system for loading.
+\item Options will be listed file system handle firmware order to maintain an established order across
+  reboots regardless of the operating system chosen for loading.
 \item Custom entries, tools, and system entries will be added after all other options.
-\item Auxiliary options will only show upon entering ``Advanced Mode'' in the picker (usually by pressing ``Space'').
+\item Auxiliary options will only be displayed upon entering ``Extended Mode'' in the boot picker (typically by pressing the \texttt{Space} key).
```

Comment:
Hrm. I am not sure I like the Advanced → Extended mode change here. Advanced mode is common in BIOS prefs and other places. Better to keep it unless there is a good reason.

Reply:
Maintained "Extended"